troposphere/static/js/components/projects/InstanceDetailsPage.jsx

```diff
@@ -36,8 +39,9 @@ export default React.createClass({
     },
 
     render() {
-        let project = stores.ProjectStore.get(Number(this.getParams().projectId));
-        let instance = stores.InstanceStore.get(Number(this.getParams().instanceId));
+        let projectId = this.context.projectId;
+        let project = stores.ProjectStore.get(projectId);
+        let instance = stores.InstanceStore.get(projectId);
```

Comment:
This is a case where we have a Route Component (aka, a component participating in a route definition), so we can use `this.props.params.{projectId,instanceId}`.